### PR TITLE
Using smaller routes

### PR DESCRIPTION
**Pull request by sebastienros**
Plaintext
Before: 8,455,689 
After: 8,527,662 

3 runs

### CODE REVIEW

File: src/BenchmarksApps/Kestrel/PlatformBenchmarks/benchmarks.plaintext.json [hunk 1]
```diff
@@ -14,14 +14,14 @@
     "Port": 8080
   },
   "PlaintextPlatform": {
-    "Path": "/plaintext"
+    "Path": "/p"
```
Comment (rynowak) [inline src/BenchmarksApps/Kestrel/PlatformBenchmarks/benchmarks.plaintext.json:17]: lol how is this not cheating?

Comment (sebastienros) [reply src/BenchmarksApps/Kestrel/PlatformBenchmarks/benchmarks.plaintext.json:17]: Exactly. I filed an issue on TE to clarify the requirements.

**Comment by sebastienros** [comment]
With preamble:
Json: 942,135
Plaintext: 8,758,381

**Comment by benaadams** [comment]
This will now be a sync with TE https://github.com/TechEmpower/FrameworkBenchmarks/pull/5579 [merged]

File: src/BenchmarksApps/Kestrel/PlatformBenchmarks/benchmarks.plaintext.json (hunk 1 above)
Comment (benaadams) [reply src/BenchmarksApps/Kestrel/PlatformBenchmarks/benchmarks.plaintext.json:17]: Not invalid for current round, may be for next https://github.com/TechEmpower/FrameworkBenchmarks/issues/5582#issuecomment-609895369

**Comment by davidfowl** [comment]
I don't like this change... Feels wrong.

**Comment by sebastienros** [comment]
@davidfowl is it the new route or the preamble that you don't like ?

**Comment by davidfowl** [comment]
😢

**Comment by benaadams** [comment]
> 😢

Baselines `aspnet/Benchmarks` so any improvements to code quality or performance are measured from where we are rather than from a very diverged old fork.

**Comment by davidfowl** [comment]
The /p and /j are BS improvements though. It doesn't feel right and it's something that can be easily rejected by TE if they decided to change that rule.

**Comment by benaadams** [comment]
> The /p and /j are BS improvements though

Completely agreed; and @sebastienros has raised it https://github.com/TechEmpower/FrameworkBenchmarks/issues/5582; it is valid for this round however and pv, actrix etc are currently doing it, it is likely not be valid for next round:

> @sebastienros I did merge this. I'd like to see the impact as well. After discussing internally, we're going to make the routes required after this round. I'll make a change to the rules here shortly for round 20 and beyond.

However the current round is being measured soon so seems wrong to drop down the pack due to a technicality? 🤷‍♂️

**Pull request merged**